baseclass/README.md

```diff
@@ -0,0 +1,50 @@
+# Apigee Java Callouts Base Class
+
+This class contains common logic for use in creating Apigee Java Callouts.
+
+For more information about Java Callouts, visit: https://docs.apigee.com/api-platform/reference/policies/java-callout-policy
+
+## Usage
+
+This github repository contains sample Java callout source code which shows how this base class can be used.
+
+To use these functions, simply extend the base class in your Java callout.
+
+## Method Summary
+
+These are the methods currently included with this base class:
+
+
+#### getOptionalProperty
```

Comment:
Nit: the formatting and line breaking in this section seems inconsistent; could we fix this?